Gopkg.toml

```diff
@@ -64,23 +64,23 @@ ignored = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.12.0-beta.1"
+  branch = "release-1.12"
```

Comment:
To guarantee all repos are in sync and compatible at e.g. VERSION=1.12.0-rc.2, use the same tag "kubernetes-$VERSION" for all k8s.io/* repos and tag "v-$VERSION" for k8s.io/kubernetes. The repos will always follow this naming scheme and have the same tags e.g. "kubernetes-1.12.0-rc.2" available at the same time